### DISCUSSION TITLE
SQLServer  Index comment

### DISCUSSION THREAD

**Discussion by EurekaJiang** [dbeaver/dbeaver#18057] [Q&A]
I found that the  "Object description (comment)" seems doesn't work in DBeaver,
I got the index comment by use the sql sentences ,but cannot see the comment  on the Indexes "Properties" view
SQL Control:
![%`I(ECCZ3({~EBXQ `MVUDY](https://user-images.githubusercontent.com/41953750/198293871-56b94938-cd8e-4e34-8bb6-69fd02a366a9.jpg)
 Indexes "Properties":
![I4T53~PX1L(VUNHKK{OIQPP](https://user-images.githubusercontent.com/41953750/198293960-4b9edc98-3fb2-4d23-b019-7c58a5133914.jpg)

**Comment by mkgrgis** [comment]
Please provide DBMS name and version...

**Comment by LonwoLonwo** [comment]
Hello @Jtworld0924 
This feature is not supported for SQL Server yet. You can create a feature request ticket for it.
